tests/Knp/Menu/Tests/Loader/ArrayLoaderTest.php

```diff
@@ -95,6 +95,9 @@ public function testSupports($data, bool $expected): void
         $this->assertSame($expected, $loader->supports($data));
     }
 
+    /**
+     * @return array<int, array<int, mixed>>
```

Comment:
I would not bother doing precise phpdoc for tests IMO

Reply:
But if we don't do it, phpstan would complain (unless putting everything in baseline, of course)

Reply:
I'm wondering about the cost/benefit ratio of applying static analysis on the tests folder...

Reply:
well, the cost is already spent... so I guess that it would cost us more to remove it

Reply:
it is spent at level 6. but your answer tells me that most of the cost at level max is also in tests. I'd rather analyse only src at max level than lowering the level due to tests.
And this applies also to future changes in tests

Reply:
I see. Considering this, I have to agree with you.